doc/suiterc.tex

```diff
@@ -751,6 +751,28 @@ \subsection{[scheduling]}
     \item {\em default:} 3
 \end{myitemize}
 
+\subsubsection[spawn to max active cycle points]{[scheduling] \textrightarrow
+ spawn to max active cycle points}
+\label{spawn to max active cycle points}
+
+Allows tasks to spawn out to \lstinline=max active cycle points=
+(\ref{max active cycle points}), removing restriction that a task has to have
+submitted before its successor can be spawned.
+
+{\em Important:} This should be used with care given the potential impact of
+additional task proxies both in terms of memory and cpu for the cylc daemon as
+well as overheads in rendering all the additional tasks in gcylc. Also, use
+of the setting may highlight any deficiencies in graphing dependent on the
```

Comment:
Can you make it clearer that this is a suite design error, i.e. that the suite writer has implicitly relied on the spawn-on-submit behaviour.  "Deficiencies in graphing" sounds like it could be another reference to rendering overheads or something (for the extra task proxies).